integration-healthcheck/src/main/java/org/symphonyoss/integration/healthcheck/CustomHealthEndpoint.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2016-2017 Symphony Integrations - Symphony LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.symphonyoss.integration.healthcheck;
+
+import static org.symphonyoss.integration.healthcheck.application.ApplicationsHealthIndicator.APPLICATIONS;
+import static org.symphonyoss.integration.healthcheck.connectivity.ConnectivityHealthIndicator.CONNECTIVITY;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.endpoint.HealthEndpoint;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.stereotype.Component;
+import org.symphonyoss.integration.healthcheck.application.ApplicationsHealthIndicator;
+import org.symphonyoss.integration.healthcheck.connectivity.ConnectivityHealthIndicator;
+
+import java.util.Collections;
+
+/**
+ * Customized health endpoint to aggregate the information about the current deployed
+ * applications and connectivity indicators.
+ * Created by rsanchez on 17/01/17.
+ */
+@Component
+public class CustomHealthEndpoint extends HealthEndpoint {
+
+  private AsyncCompositeHealthIndicator healthIndicator;
+
+  @Autowired
+  public CustomHealthEndpoint(IntegrationBridgeHealthAggregator healthAggregator,
+      ApplicationsHealthIndicator applicationsHealthIndicator,
+      ConnectivityHealthIndicator connectivityHealthIndicator) {
+    super(healthAggregator, Collections.<String, HealthIndicator>emptyMap());
+
+    this.healthIndicator = new AsyncCompositeHealthIndicator(healthAggregator);
```

Comment:
Using "new" here makes this class untestable with mocks.  At a minimum, you should have a constructor that allows injecting a HealthIndicator.  Does this class have a unit test.  Lemme check....  Nope.